runtime/queries/bicep/highlights.scm

```diff
@@ -0,0 +1,71 @@
+; Keywords
+
+[
+  "module"
+  "var"
+  "param"
+  "import"
+  "resource"
+  "existing"
+  "if"
+  "targetScope"
+  "output"
+] @keyword
+
+; Functions
+
+(decorator) @function.builtin
+(functionCall) @function.call
+(functionCall
+  (functionArgument
+    (variableAccess) @variable))
+
+; Literals/Types
+
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+(resourceDeclaration
+  (string
+    (stringLiteral) @string.special))
+
+(moduleDeclaration
+  (string
+    (stringLiteral) @string.special))
+
+[
+  (string)
+  (stringLiteral)
+] @string
+
+(nullLiteral) @keyword
+(booleanLiteral) @constant.builtin.boolean
+(integerLiteral) @constant.numeric.integer
+(comment) @comment
+
+(string
+  (variableAccess
+    (identifier) @variable))
+
+(type) @type
+
+; Variables
+
+(localVariable) @variable
+
+; Statements
+
+(object
+  (objectProperty
+    (identifier) @identifier))
+
+(propertyAccess
+  (identifier) @identifier)
+  
+(ifCondition) @conditional
```

Comment:
```suggestion
(ifCondition) @keyword.control.conditional
```

Reply:
Thanks! Do you have a good resource/reference for available captures?

Reply:
https://docs.helix-editor.com/master/themes.html#syntax-highlighting